android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -119,6 +126,26 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         });
     }
 
+    private void createTimePicker() {
+        Calendar calendar = Calendar.getInstance();
+        int currentHour = calendar.get(Calendar.HOUR_OF_DAY);
+        int currentMinutes = calendar.get(Calendar.MINUTE);
+
+        timePicker = new MaterialTimePicker.Builder()
+                .setInputMode(MaterialTimePicker.INPUT_MODE_KEYBOARD)
```

Comment:
the problem is that i can't test it in this view.

Reply:
Okay, but can you than change the name of the buttons to german?
Its not critical, but it would be nice.
otherwise this looks good to me.

Reply:
I answered to the wrong change requests :D

@Cebox82 I tested it and now it will work now with 

```java
     timePicker = new MaterialTimePicker.Builder()
                .setInputMode(MaterialTimePicker.INPUT_MODE_CLOCK)
```

So there is no need for a bigger change